TestingTravis/TestClass.swift

```diff
@@ -11,9 +11,13 @@ import Foundation
 class TestClass: NSObject {
     
     var forceUnwrapMe: String?
+    var forceUnwrap2: Int?
     
     override init() {
         print(forceUnwrapMe!)
+        print(forceUnwrap2!)
+        
+        let unused = 0
     }
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/b9c109c0003830a07d488f9b1648f299ae45e9b0/TestingTravis/TestClass.swift#L22"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:22</code></td>
    </tr>
  </tbody>
</table>